src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -412,10 +412,11 @@ private DotnetHost DotnetHostInfo
                 {
                     CommandLineBuilderExtension commandLineBuilder = new CommandLineBuilderExtension();
                     AddCommandLineCommands(commandLineBuilder);
+                    var commandLine = commandLineBuilder.ToString();
 
                     // ToolExe delegates back to ToolName if the override is not
                     // set.
-                    // So, we can't check if it's unset, as that will recurse
+                    // So, we can't check if it's unset, as that will recur
```

Comment:
When it comes to not knowing English, I'm never late to the party, I was there before you. 😄